growthstreet_zoo/zoo.py

```diff
@@ -124,6 +115,8 @@ def is_eaten(self, animal):
         """Return a Boolean indicating whether an instance of an animal is eaten."""
         self.prey = self.COMPETITION_MAP[self.species]
         if animal.species in self.prey:
+            animal.status = animal.STATUS_DEAD
```

Comment:
You've changed what this method does, but not the docstring.  Also I think is_eaten() is no longer the right name, since it implies it's just getting some information, not performing an action.